### PR TITLE
Update components: sb-textfield, sb-avatar-group and sb-link

### PR DESCRIPTION
**Pull request by emanuelgsouza**
This PR implement the following new features:
* Add support to render textarea passing the `type="textarea"` to component and showing the characters remaining using the maxlength attribute.
* Add `darkBg` property to `SbAvatarGroup` component.
* Add `variant` property to `SbLink` component.
* Add a bunch of new icons.

### CODE REVIEW

File: src/components/TextField/SbTextField.vue [hunk 1]
```diff
@@ -75,31 +115,82 @@ export default {
...
+    },
+
+    isAlmostRaiseMaxlength() {
+      const limit = Math.floor(this.maxlengthParsed * 0.2)
+
+      return this.computedValueLength >= this.maxlengthParsed - limit
+    },
+
+    computedValueLength() {
+      if (typeof this.computedValue === 'string') {
```
Comment (thiagosaife) [inline src/components/TextField/SbTextField.vue:176]: Não poderia fazer a coerção pra string direto? 
`return \`${this.computedValue}.length\``

**Review by ademarCardoso — APPROVED** [review]
Nice work man!

File: src/components/TextField/SbTextField.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (emanuelgsouza) [reply src/components/TextField/SbTextField.vue:176]: Good! That's make sense.

**Review by fariacc — APPROVED** [review]

**Review by fariacc — APPROVED** [review]

**Review by ademarCardoso — APPROVED** [review]

**Review by gustavomelki — APPROVED** [review]
Excellent work as always!

**Review by thiagosaife — APPROVED** [review]
Nice!

**Pull request merged**